src/gatsby/utils/fetchCache.ts

```diff
@@ -1,3 +1,32 @@
+/**
+ * For some reason we sometimes see errors fetching from our registry, because
```

Comment:
Brought back the retry logic... it does seem like we actually need it for some reason?